test/unit/translators/twitter_translators/tweet_translator_test.rb

```diff
@@ -51,7 +51,7 @@ def build_tweet
     end
   end
 
-  test "no new noise when importing old tweet" do
+  test 'no new noise when importing old tweet' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.